doc: Fix translation procedure in i18n.rst (#1407)
groonga/groonga: doc/source/contribution/documentation/i18n.rst

1. Install Sphinx, if it is not installed.
2. Clone Groonga repository.
3. Update .edit files
4. Edit .edit files
5. Update .po files.
6. Generate HTML files.
7. Confirm HTML output.

> 8. Repeat 3.-7. until you get good result.

Reviewer: ```suggestion
8. Repeat 3.-7. until you finish to translate.
```

to align with the previous wording.

Author: Ah, maybe you see the old diff. It had been fixed in https://github.com/groonga/groonga/pull/1407/commits/5bd6dc43776288a58f362287a91a2432bfd6c3cf (the latest commit).

If you have troubles on the above steps, you can use source
files available on https://packages.groonga.org/source/groonga/ .

How to update .edit files
-------------------------

You can update .edit files by running `make` on
doc/locale/${LANGUAGE}/LC_MESSAGES. (Please replace
`${LANGUAGE}` with your language code such as 'ja'.)::

  % cd doc/locale/ja/LC_MESSAGES
  % make

How to edit .edit
-----------------

> .edit is a alias of .po, so .edit files are actually .po files. 
> .edit files are for editing and .po files are for versioning.

Reviewer 2: ```suggestion
Both of ``.edit`` files and ``.po`` files are `PO files <https://www.gnu.org/software/gettext/manual/html_node/PO-Files.html>`__. ``.edit`` files are for editing and ``.po`` files are for versioning.
```

.. _Poedit: http://www.poedit.net/
.. _gted: http://www.gted.org/

How to update .po files
-----------------------

> After editing .edit files, you can update .po files by running `make` on

Reviewer 2: ```suggestion
After editing .edit files, you can update .po files by running ``make`` on
```

> doc/locale/${LANGUAGE}/LC_MESSAGES. (Please replace

Reviewer 2: ```suggestion
``doc/locale/${LANGUAGE}/LC_MESSAGES``. (Please replace
```

`${LANGUAGE}` with your language code such as 'ja'.)::

  % cd doc/locale/ja/LC_MESSAGES

>   % make

Reviewer 2: Could you use `code-block`?

Author: Fixed
